constants/user-permission-constants.js

```diff
@@ -28,7 +29,7 @@ module.exports = Object.freeze({
         CREATE: "submission_request:create",
         SUBMIT: "submission_request:submit",
         REVIEW: "submission_request:review",
-        DELETE: "submission_request:delete",
+        DELETE: "submission_request:cancel",
```

Comment:
Can you rename the constant name from DELETE to CANCEL, and update any places use it? We usually don't want to rename constants (only change value) but in this case, we want to rename, because we might need a real DELETE permission in the future.